lms/static/js/student_account/AccountsClient.js

```diff
@@ -0,0 +1,23 @@
+import 'url-search-params-polyfill';
+import 'whatwg-fetch';
+import Cookies from 'js-cookie';
+
+const deactivate = (password) => fetch('/api/user/v1/accounts/deactivate_logout/', {
+  method: 'POST',
+  credentials: 'same-origin',
+  headers: {
+    'Content-Type': 'application/x-www-form-urlencoded',
+    'X-CSRFToken': Cookies.get('csrftoken'),
+  },
+  body: new URLSearchParams({ password }),
+}).then((response) => {
+  if (response.ok) {
+    return response;
+  }
+
+  throw new Error(response);
+});
+
+export {
+  deactivate,
+};
```

Comment:
Nitpick: missing new line.

Reply:
![image](https://user-images.githubusercontent.com/8136030/39486049-d712deee-4d48-11e8-9305-1e9abaaf1a84.png)

Odd, it's definitely there and saving the file doesn't result in any changes.

🤷‍♂️